jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/HandlerRegistry.java

```diff
@@ -28,16 +31,18 @@
 import java.util.Map;
 import java.util.function.Supplier;
 
-final class HandlerRegistry {
+public final class HandlerRegistry {
   private static final String INSTRUMENTATION_NAME = "io.opentelemetry.contrib.jfr";
   private static final String INSTRUMENTATION_VERSION = "1.7.0-SNAPSHOT";
 
   private final List<RecordedEventHandler> mappers;
 
+  public static HashSet<String> garbageCollectors = new HashSet<String>();
```

Comment:
A public static mutable constant is a code smell. Let's get rid of this and instead pass the names of the garbage collectors as an constructor argument to `OldGarbageCollectionHandler`.

Reply:
done!